lib/DDG/Spice/Untappd.pm

```diff
@@ -0,0 +1,24 @@
+package DDG::Spice::Untappd;
+
+# ABSTRACT: Discover your favorite beers
+
+use DDG::Spice;
+
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d";
+
+spice wrap_jsonp_callback => 1;
+
+spice to => 'https://api.untappd.com/v4/search/beer?' .
+                'client_id={{ENV{DDG_SPICE_UNTAPPD_CLIENT_ID}}}&' .
+                'client_secret={{ENV{DDG_SPICE_UNTAPPD_CLIENT_SECRET}}}&' .
+                'q=$1';
+
+triggers any => 'untappd', 'beer', 'beers', 'brewery', 'breweries';
+
+handle remainder => sub {
+    return $_ if $_;
```

Comment:
@urbanserj what kinds of queries exactly are you expecting to trigger this Instant Answer?

Right now, any query the contains one of your trigger words, anywhere in the string, is going to try and fire this Instant Answer. I suspect the majority of all those queries will be irrelevant to this API?